### PR TITLE
clarify these examples are for 2D images where NAXIS=2

### PR DESCRIPTION
**Pull request by teuben**
simple doc change, but since it bit me when I happen to work on a 4D image cube, I'm proposing this clarification.

### CODE REVIEW

**Comment by nden** [comment]
@teuben I can see now how this can be confusing. Thanks for the clarifications. I'd say there's no need for a changelog entry but  if someone objects please remove the label and ask for the change.

File: docs/wcs/examples/from_file.py [hunk 1]
```diff
@@ -21,7 +21,8 @@ def load_wcs_from_file(filename):
     # Print out all of the settings that were parsed from the header
     w.wcs.print_contents()
 
-    # Some pixel coordinates of interest.
+    # Four pixel coordinates of interest.
```
Comment (pllim) [inline docs/wcs/examples/from_file.py]: I only see three pixels defined. Am I missing something?

Comment (teuben) [inline docs/wcs/examples/from_file.py]: my bad, it is three.  should I update the pull request?

Comment (pllim) [inline docs/wcs/examples/from_file.py]: Yes, please do.

**Pull request merged**